src/components/data-exploration/heatmap/HeatmapGroupBySettings.jsx

```diff
@@ -13,7 +13,7 @@ import PropTypes from 'prop-types';
 import _ from 'lodash';
 import { updatePlotConfig } from '../../../redux/actions/componentConfig';
 import ReorderableList from '../../ReorderableList';
-import { getCellSets } from '../../../redux/selectors';
+import { getCellSets, getCellSetsHierarchy } from '../../../redux/selectors';
```

Comment:
do you need `getCellSets`? are you using anything else from cellSets that isn't obtained from `getCellSetsHierarchy`?

Reply:
and this same thing in the other files that use `getCellSetsHierarchy`